Token.js

```diff
@@ -2372,14 +2373,15 @@ class Token {
 		// HEALTH AURA / DEAD CROSS
 		selector = "div[data-id='" + this.options.id + "']";
 		let token = $("#tokens").find(selector);
-		this.build_stats(token)
-		this.toggle_stats(token)
-		this.update_health_aura(token)
-		this.update_dead_cross(token)
-		// this.toggle_player_owned(token)
-		toggle_player_selectable(this, token)
-		//check_token_visibility(); // CHECK FOG OF WAR VISIBILITY OF TOKEN
-		debounceLightChecks();
+		let promises = [];
+		promises.push(new Promise(() => setTimeout(this.build_stats(token), 100)));
+		promises.push(new Promise(() => setTimeout(this.toggle_stats(token), 100)));
+		promises.push(new Promise(() => setTimeout(this.update_health_aura(token), 100)));
+		promises.push(new Promise(() => setTimeout(this.update_dead_cross(token), 100)));
+		promises.push(new Promise(() => setTimeout(toggle_player_selectable(this, token), 100)));
+		promises.push(new Promise(() => setTimeout(debounceLightChecks(), 100)));
```

Comment:
Can you explain the benefit of making these timeouts?

Reply:
Was more just to keep it flowing for the user. I think the best bet would to just background debounce them. Ill play with background tasks and just debounce the ones not debounced already